### ISSUE TITLE
Duration B without Manned Orbital does not carry a milestone penalty

### ISSUE THREAD

**Issue by hweimer** [raceintospace/raceintospace#408]
While I like the new duration system much better than the original one, there is still one case that feels a bit odd. Now, after flying a manned suborbital, you can immediately go for duration B without having to do a duration A manned orbital first. I can see why this is the case, as the suborbital has already completed the duration level A achievement, but it is somewhat strange to put the first person in orbit there for 3-5 days. Historically, the first duration B mission was Vostok 3 (3d22h28m), after Vostok 2 (1d1h18m) and Vostok 1 (1h48m).

**Comment by peyre** [comment]
True, but the game uses a straight step system, for lack of a better word: anything up to 3 days is Dur A, etc., so anything within a duration level is abstracted out.  I suppose we could make a special case for going from Suborbital to Orbital Duration: one elegant solution would be to make the Orbital a milestone for the Orbital Duration.  I don't know what's involved in adding a milestone step though.

**Comment by rnyoakum** [comment]
It would be possible to modify the Duration system so that it would apply the penalty for skipping Duration A if the player hasn't achieved Milestone_EarthOrbit. That probably wouldn't be hard.

**Comment by peyre** [comment]
I was just thinking about this issue.  As it is, if you've flown a suborbital, that means (by the game's logic) you have flown a Duration A mission, so you're not skipping a duration step.  However, if you schedule a Dur B mission without having flown a suborbital, you get a duration penalty.

**Comment by hweimer** [comment]
How was this in the original game? IIRC, you got a -5 penalty on the first duration mission, but no further penalties on any later duration mission (as long as they weren't milestone missions like Manned Lunar Pass). Then, you would normally fly Sub Orbital -> Earth Orbital -> Dur B -> Dur D, which is the same number of missions as the currently allowed Sub Orbital -> Dur B -> Dur C -> Dur D. If the change to the duration system did not change the number of manned missions you need to fly, then this is probably not an issue. After all, it's pretty obvious that you fly way less manned missions in RIS than there were in reality.

**Comment by peyre** [comment]
Well...and I just checked this against the CD version...original BARIS had a bug that would always apply a duration penalty to your first Duration mission.  However it wasn't _supposed_ to apply a penalty at all, and I notice it doesn't apply a milestone penalty.  So, the intended behavior would have let you go straight from the Suborbital to the Orbital Duration.  That doesn't mean we can't change that though.  I think it's a good idea.

**Comment by peyre** [comment]
Funny...I've always been a little disappointed there wasn't more you could do with a Mercury than a Suborbital, Orbital, EVA, and Dur B.  (_And_ the EVA probably shouldn't be there, realistically, but that's another discussion.)  I don't have a strong opinion on this one - if you think it's best left alone I'm ok with that.

**Comment by hweimer** [comment]
My impression is that the limitations in terms of what you can do with Mercury/Vostok is also a reflection of the thought that you probably don't want to do a lot with them. Personally, I like the way the game conveys the hotshot mentality of the early space program (although "statistical death trap" is also a valid characterization), and it would be sad (and historically inaccurate) if serious play would demand abandoning that and go for Gemini/Voskhod right away. Currently, I think that developing the one-seater is worth doing, but changes like that (or #393) might tilt the balance.

**Comment by peyre** [comment]
Yes, my only real concern is that if we were to take EVA away from Mercury/Vostok, we should add another prestige mission that it could fly - but which you could do with a different capsule if you want to.
